.travis/install_cython.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
 # grep setup.py for the pinned version of cython
-PINNED_CYTHON=$(grep -oE 'cython==[0-9]+\.[0-9]+\.[0-9]+' setup.py)
+PINNED_CYTHON=$(grep -oE 'cython>=[0-9]+\.[0-9]+\.[0-9]+,<[0-9]+\.[0-9]+\.[0-9]+' setup.py)
```

Comment:
Subsuituting `[0-9]+\.[0-9]+\.[0-9]+` with `x`, this becomes

`cython>=x,<x`

which I think will not match anything?

Reply:
It works for me:

```
$ git branch 
  master
* update-cython
$ grep -oE 'cython>=[0-9]+\.[0-9]+\.[0-9]+,<[0-9]+\.[0-9]+\.[0-9]+' setup.py
cython>=0.29.1,<1.0.0
```

Reply:
Oh, sorry, I was not reading well. Ignore me.

Reply:
I believe this needs quotes wrapped around it - the statement is:
```$ pip install cython>=0.29.1,<1.0.0```
But I think it needs to be
```$ pip install "cython>=0.29.1,<1.0.0"```

I just ran into this locally - the latest version is `0.29.6`, but Travis seems to be using `0.29.1` still - should that be getting updated to the latest available version on each build?  I though these test containers were rebuild each time, should this instead be

```$ pip install "cython>=0.29.1,<1.0.0" --upgrade```

Reply:
(happy to make that change myself - trying to figure out what the appropriate path is)

Reply:
False alarm about the second thing there - travis _is_ correctly using the latest version: https://github.com/dimagi/jsonobject/pull/165#issuecomment-476258567

Reply:
@esoergel travis handles the install using this script:

https://github.com/dimagi/jsonobject/blob/e12762ac6e12d041f7d262787f1656fe2cecb599/.travis/install_cython.sh

Would your suggestion make this script obsolete?

Reply:
no, I don't think so - looks like it works fine in the context of the script - I'd tried copy/pasting it to my terminal and I got an error because the version arg wasn't quoted, but if I run this script directly, it works fine.